boards/nordic/nrf54l15dk/nrf54l15_cpuapp_common.dtsi

```diff
@@ -135,7 +135,7 @@
 
 	mx25r64: mx25r6435f@0 {
 		compatible = "jedec,spi-nor";
-		status = "okay";
+		status = "disabled";
```

Comment:
This is not correct

Reply:
https://github.com/mcu-tools/mcuboot/commit/242db1a9301dc21bd6d45351987ee80b48a86790 needs to be brought in as fromtree

Reply:
No other DK has this, the pdk is wrong

Reply:
nrf52840dk, nrf5340dk, thingy53 - disabled(empty), nrf9161dk , nrf9151dk, nrf9131dk - disabled

Reply:
Not sure how you got that
> nrf52840dk, nrf5340dk, thingy53 - disabled(empty)

Incorrect, enabled on all these boards

> nrf9161dk , nrf9151dk, nrf9131dk - disabled

Correct due to nrf9160dk having it disabled because the original dk did not have it, it was disabled in the revised hardware to maintain compatibility with the original, the other dks should have had it enabled when they copied the base board

Reply:
I see that for the first set, (qspi memories, compatible = "nordic,qspi-nor";) status is empty.

Reply:
No status means okay, it inherits from the parent, it would need an explicit disabled to be disabled

Reply:
I have to disable spi-nor driver thought Kconfig then (in a new mcuboot patch). @nordicjm Will be that fine?

Reply:
For which board? nrf54l15dk is already disabled, if you mean nrf52840dk or nrf5340dk then they can be submitted upstream if needed, thingy53 should not be disabled because the secondary slot is in QSPI

Reply:
my mistake

Reply:
@nordicjm Can you approve?